src/frontend/komponenter/Visningskomponenter/Tekster.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { forwardRef } from 'react';
```

Comment:
Syntes ikke denne filen er nødvendig. Vi har alt "jobbet hardt" for å ikke ta den inn